pkg/volume/quobyte/quobyte.go

```diff
@@ -281,8 +285,8 @@ func (quobyteVolume *quobyte) GetPath() string {
 
 	// Quobyte has only one mount in the PluginDir where all Volumes are mounted
```

Comment:
This comment should be adjusted since all Quobyte volumes are mounted in a sub dir?

Reply:
Done